docs/Xamarin.Forms/Image.xml

```diff
@@ -53,7 +53,7 @@
         <code lang="csharp lang-csharp"><![CDATA[
 var image = new Image { Source = "picture.png" };]]></code>
       </example>
-      <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://developer.xamarin.com/samples/tag/Xamarin.Forms/" target="_blank">Sample Applications</a></format> page, has a ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
+      <para>The FormsGallery sample, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/samples/" target="_blank">Sample Applications</a></format> page, has a ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
```

Comment:
As before - these links point to both the specific example, and the more general page with all the XF samples :)
```suggestion
      <para>The <format type="text/html"><a href="https://docs.microsoft.com/samples/xamarin/xamarin-forms-samples/formsgallery/" target="_blank">FormsGallery sample</a></format>, which can be found on the <format type="text/html"><a href="https://docs.microsoft.com/samples/browse/?products=xamarin&term=Xamarin.Forms&terms=Xamarin.Forms" target="_blank">Sample Applications</a></format> page, has an ImageDemoPage.cs file. This file contains a longer and more complete example.</para>
```